gensrc/proto/segment_v2.proto

```diff
@@ -162,6 +184,12 @@ message ColumnMetaPB {
     optional uint64 num_rows = 11;
     repeated string children_column_names = 12;
 
+    // persist info for PathInData that represents path in document, e.g. JSON.
```

Comment:
add comment to explain each extracted subcolumn of variant column will generate a column in be schema

Reply:
comment already in ColumnPathInfo